src/lightmap.cpp

```diff
@@ -803,25 +803,25 @@ bool map::pl_sees( const tripoint &t, const int max_range ) const
              map_cache.sm[t.x][t.y] > 0.0 );
 }
 
-bool map::pl_line_of_sight( const tripoint &t, const int max_range ) const
-{
-    if( !inbounds( t ) ) {
-        return false;
-    }
-
-    const level_cache &map_cache = get_cache_ref( t.z );
-    if( map_cache.camera_cache[t.x][t.y] > 0.075f ) {
-        return true;
-    }
-
-    if( max_range >= 0 && square_dist( t, get_player_character().pos() ) > max_range ) {
-        // Out of range!
-        return false;
-    }
-
-    // Any epsilon > 0 is fine - it means lightmap processing visited the point
-    return map_cache.seen_cache[t.x][t.y] > 0.0f;
-}
+//bool map::pl_line_of_sight( const tripoint &t, const int max_range ) const
+//{
+//    if( !inbounds( t ) ) {
+//        return false;
+//    }
+
+//    const level_cache &map_cache = get_cache_ref( t.z );
+//    if( map_cache.camera_cache[t.x][t.y] > 0.075f ) {
+//        return true;
+//    }
+
+//    if( max_range >= 0 && square_dist( t, get_player_character().pos() ) > max_range ) {
+//        // Out of range!
+//        return false;
+//    }
+
+//    // Any epsilon > 0 is fine - it means lightmap processing visited the point
+//    return map_cache.seen_cache[t.x][t.y] > 0.0f;
+//}
 
```

Comment:
Unless you think it's worth keeping, this means `pl_line_of_sight` could be done away with entirely, rather than just commented out.
```suggestion
```
And if it's deleted, so should the corresponding header definition:
https://github.com/CleverRaven/Cataclysm-DDA/blob/20c59f6d19498477ff8a54d1b581307e7182f88b/src/map.h#L1830C9-L1836C48

Reply:
The new echolocation in #70797 uses this dead code, but I intend to switch it to the method in this PR. This is probably going to get merged first, so I guess I can just remove it and update echolocation accordingly later.